Tests.Unit.Presentation.Web/Models/V2/ItContractWriteModelMapperTest.cs

```diff
@@ -227,11 +227,12 @@ public static IEnumerable<object[]> GetUndefinedTerminationDataPropertiesInput()
             bool noNotes,
             bool noEnforceValid,
             bool noValidFrom,
-            bool noValidTo)
+            bool noValidTo,
+            bool noCriticalityTypeUuid)
         {
             //Arrange
             var input = new CreateNewContractRequestDTO();
-            ConfigureGeneralDataInputContext(noContractId, noContractTypeUuid, noContractTemplateUuid, noAgreementElementUuids, noNotes, noEnforceValid, noValidFrom, noValidTo);
+            ConfigureGeneralDataInputContext(noContractId, noContractTypeUuid, noContractTemplateUuid, noAgreementElementUuids, noNotes, noEnforceValid, noValidFrom, noValidTo, noCriticalityTypeUuid);
```

Comment:
you are changing the input in all of these tests but there are no changes to the assert blocks.

Please fix this